product_security/recommendations_for_customers/customer_logging_metrics_monitoring.adoc

```diff
@@ -0,0 +1,20 @@
+[[dedicated-security-logging-metrics-monitoring-customers]]
+= Logging, Metrics, and Monitoring
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+{product-title} administrators can view all customer application logs in the
+cluster, but application developers can only view logs for projects they have
+permission to view. Kibana can be accessed at:
+
+----
+https://logs.[cluster-name].openshift.com
```

Comment:
Add text: Audit logs are maintained by Red Hat for Compliance purposes and are not shareable with customers at this time.